### ISSUE TITLE
CaCO3 burial improvement needs to be added to CISO

### ISSUE THREAD

**Issue by mnlevy1981** [marbl-ecosys/MARBL#82]
Keith added some new features that made it to the POP trunk (pre-MARBL) in November of 2015: part of the `cesm_pop_2_1_20151117` trunk tag. This incudes 

```
introduce caco3_bury_thres_opt, default=omega_calc
```

Part of this commit is a block of code that is now in `marbl_mod`

```
          if (zw(k) < caco3_bury_thres_depth) then
             P_CaCO3%sed_loss(k) = P_CaCO3%sflux_out(k) + P_CaCO3%hflux_out(k)
          endif
```

But the similar block of code in `marbl_ciso_mod` is

```
       if (column_zw < 3300.0e2_r8) then
          flux = P_CaCO3_ciso%sflux_out(k) + P_CaCO3_ciso%hflux_out(k)
          P_CaCO3_ciso%sed_loss(k) = flux
       endif
```

(There are probably other blocks of code that also need to be updated, this was the easiest to find to provide an example). Note that this is tied to #24 because we may end up removing `zw` from MARBL (so these threshold depths would be compared to `zt`)

**Comment by klindsay28** [comment]
The ciso version of CaCO3 burial does not depend on `caco3_bury_thres_iopt`, like the base ecosys version does. I think this qualifies as a bug.

I think this discrepancy would be caught if we implemented #204.

**Comment by mnlevy1981** [comment]
Fixed in [marbl0.26.0](https://github.com/NCAR/MARBL/releases/tag/marbl0.26.0)
